go/billing/models.py

```diff
@@ -183,7 +183,7 @@ def message_credit_cost(self):
     def storage_credit_cost(self):
         """Return the calculated cost per session (in credits)."""
         return self.calculate_storage_credit_cost(
-            self.session_cost, self.markup_percent)
+            self.storage_cost, self.markup_percent)
```

Comment:
I remember this two. Possible reasons I can think of:
  - it was actually the `calculate_storage_credit_cost` line that was noticed in an earlier PR
  - the fix got lost in the merge (I hope its not that, could mean other things got lost)
  - I forgot to fix it before merging the relevant PR into this PR